cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -501,7 +503,15 @@ void SparsePoseGraph::RunOptimization() {
 std::vector<std::vector<mapping::TrajectoryNode>>
 SparsePoseGraph::GetTrajectoryNodes() {
```

Comment:
@cschuet you were referring to this PR, right? Why are you not returning a `mapping::MapById<mapping::NodeId, TrajectoryNode>` here, like for `GetAllSubmapData()`?

Reply:
No sorry this one is obsolete. It never made it beyond the WIP stage.